ZenPacks/zenoss/ZenPackLib/lib/spec/GraphDefinitionSpec.py

```diff
@@ -21,7 +24,7 @@ def __init__(
             hasSummary=None,
             graphpoints=None,
             comments=None,
-            _source_location=None
+            _source_location=None,
```

Comment:
In the other classes you added log as a parameter.  did you forget to put it in here?